src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -174,25 +162,30 @@ const getExpandedContent = (props: ContentProps) => {
   ) : null;
 };
 
-const SpeciesMainViewStats = (props: Props) => {
+const SpeciesMainViewStats = () => {
+  const dispatch = useDispatch();
+  const activeGenomeId = useSelector(getActiveGenomeId) || '';
```

Comment:
Because the selector that uses it wants activeGenomeId to be a string.

Reply:
>The reason I was reluctant to do this is because getGenomeExampleFocusObjects and getCommittedSpeciesById should not even be called if genome id is null; it doesn't make sense. But with stupid React hooks, you can't call them conditionally; so it's the choice between hacky and ugly 😞

The ugly side of hooks we didn't come across before 😄